README.md

```diff
@@ -171,15 +171,15 @@ danm        **fakeipam**      host-device  ipvlan       macvlan      ptp
 
  **7. Create the netwatcher DaemonSet by executing the following command from the project's root directory:**
  ```
-kubectl create -f integration/manifests/netwatcher/netwatcher_ds.yaml
+kubectl create -f integration/manifests/netwatcher/
 ```
 Note: don't forget to change the names of files and directories pointing to valid kubeconfig files, and TLS certificates used by the K8s API server in your infrastructure before instantiating the component! 
```

Comment:
we should probably update other references throughout the document implying kubeconfig based setup

or better, we should describe somewhere that on the code-level both kubeconfig based and RBAC / SA based incluster configuration is supported, but the example describes the latter

Reply:
on a second thought: I'm also fine supporting only RBAC fro netwatcher too. Anyway this should be done for production environment in most cases
but then we should delete the kubeconf option from the netwatcher code

Reply:
Either is fine by me, you choose 😄. However if we remove the kubeconfig option, we break BW compatibility. It might not hurt anybody, but still.

Reply:
yeah it's BW incompatible, but also the old solution is not good anyway

I think RBAC is "default" enough that we can do it